guide/miscellaneous/useful-packages.md

```diff
@@ -1,35 +1,35 @@
 # Useful packages
 
-## moment.js
+## Day.js
 
 ::: tip
-Official documentation: [https://momentjs.com/docs](https://momentjs.com/docs)
+Official documentation: [https://day.js.org/](https://day.js.org/en)
 :::
 
-Moment is a powerful package for working with dates in JavaScript.  
+Day.js is a powerful package that parses, validates, manipulates, and displays dates and times in JavaScript.
 It allows you to quickly and easily format dates in any way you want or parse strings back into JavaScript Date objects.  
-There are even some extensions for it to allow you to work with durations and more.
+There are even some plugins for it to allow you to work with durations and more.
```

Comment:
Day.js without any plugin is poor on features, so you should say that it often requires plugins.

Reply:
Surely a better option would be to use a lib such as date-fns? Obviously, there are many different date-time libraries, but I've personally found date-fns to be one of the better ones.

Reply:
Had talked with @Danktuary regarding this earlier, and we decided to use `Day.js` in the guide.

Reply:
For the purposes and examples listed in this page, I think day.js is fine. It has a simple API, is small in size, and has plugins for common use cases.